### PR TITLE
Upgrade babel-plugin-inline-replace-variables to 1.3.1

### PR DESCRIPTION
**Pull request by alexBaizeau**
Also added some tests for the actual feature

### CODE REVIEW

**Comment by alexBaizeau** [comment]
The test failures seems irrelevant ember-cli 1.12.2 does not know about node 8 that just came out
```
ember-cli@2.12.1: The engine "node" is incompatible with this module. Expected version "^4.5 || 6.* || 7.*".
```

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 language: node_js
 node_js:
-  - "node"
+  - "7"
```
Comment (minichate) [inline .travis.yml]: Thanks!

**Comment by minichate** [comment]
Published as `1.0.4`

**Comment by alexBaizeau** [comment]
Thank you!

**Pull request merged**